movie_hub.service

```
@@ -0,0 +1,10 @@
+[Unit]
+Description=TP project kinopoisk by vkladishy
```

Я бы в Unit еще добавил требование на наличие сети

```suggestion
Description=TP project kinopoisk by vkladishy
Wants=network-online.target
After=network.target network-online.target
```

Таким образом сервис будет запускаться только после успешного поднятия сети и установления соединения с интернетом